packages/drivers/routerlicious-driver/src/test/mapWithExpiration.spec.ts

```diff
@@ -287,7 +287,7 @@ describe("MapWithExpiration", () => {
 					map.set(1, "one");
 					map.forEach(() => {
 						assert.notEqual(
-							this,
+							globalThis,
```

Comment:
As test for notEqual is very easy to make pass with any change. Is it possible to make a product change that breaks this? So, we can be sure the right thing is tested.?

Reply:
[AB#7431](https://dev.azure.com/fluidframework/internal/_workitems/edit/7431)